generators/server/templates/_pom.xml

```diff
@@ -974,9 +977,6 @@
                     Enable the line below to have remote debugging of your application on port 5005
                     <jvmArguments>-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005</jvmArguments>
                     -->
-                    <arguments>
```

Comment:
To test the issue I said above generate an app with H2 as dev DB and mysql as prod DB. Dont start mySQL so that prod should fail. Now run `gradlew -Pprod` or maven equivalent and app starts up and log says prod profile but DB seems to work somehow. and ofcourse if you check there is a H2 DB running. Now add `--spring.profiles.active=prod` as argument to maven/gradle build and try again to boot prod profile. App fails to start as mySql is not running.